packages/react-components/react-overflow/src/Overflow.cy.tsx

```diff
@@ -19,25 +20,36 @@ const selectors = {
   menu: 'data-test-menu',
 };
 
-const Container: React.FC<{ children?: React.ReactNode; width?: number } & Omit<OverflowProps, 'children'>> = ({
-  children,
-  width,
-  ...overflowProps
-}) => {
+const Container: React.FC<
+  { children?: React.ReactNode; width?: number; overflowAxis?: OverflowAxis } & Omit<OverflowProps, 'children'>
```

Comment:
```suggestion
  { children?: React.ReactNode; width?: number } & Omit<OverflowProps, 'children'>
```

`overflowAxis` is under `OverflowProps`